src/key.js

```diff
@@ -771,6 +771,84 @@ Key.prototype.verifyAllUsers = async function(keys) {
   return results;
 };
 
+/**
+ * generate a new subkey to a Key
+ * @param  {Object} options   (optional) options for creating subkey, e.g. {sign: true, passphrase: '123', algorithm:..., curve:...}
```

Comment:
Could you please enumerate all of the options that are possible in this object in the documentation?

Reply:
no problem.